client/a8c-for-agencies/sections/partner-directory/dashboard/index.tsx

```diff
@@ -46,31 +95,25 @@ export default function PartnerDirectoryDashboard() {
 		document.querySelector( '.partner-directory__body' )?.scrollTo( 0, 0 );
 	}, [] );
 
-	const isSubmitted = true; // FIXME: Replace with actual value
-	const brandStatuses = [
-		{
-			brand: 'WordPress.com',
-			status: 'Approved',
-			type: 'success',
-		},
-		{
-			brand: 'WooCommerce.com',
-			status: 'Not approved',
-			type: 'error',
-		},
-		{
-			brand: 'Pressable.com',
-			status: 'Pending',
-			type: 'warning',
-		},
-		{
-			brand: 'Jetpack.com',
-			status: 'Pending',
-			type: 'warning',
-		},
-	] as { brand: string; status: string; type: BadgeType }[]; // FIXME: Replace with actual value
+	const { availableProducts } = useFormSelectors();
+
+	const isSubmitted =
+		applicationData?.status === 'pending' || applicationData?.status === 'in-progress';
+	const isCompleted = applicationData?.status === 'completed';
 
-	const isCompleted = true; // FIXME: Replace with actual value
+	const directoryApplicationStatuses: DirectoryApplicationStatus[] = [];
```

Comment:
Just a small nitpick, but you could probably use `applicationData?.directories?.reduce()` function here.

Reply:
https://github.com/Automattic/wp-calypso/pull/91769/commits/b6d8589037fa635e5d782e14839c8131fbdc04df